src/state/slices/opportunitiesSlice/types.ts

```diff
@@ -12,6 +12,7 @@ export type OpportunityMetadata = {
   tvl: string
   type: DefiType
   underlyingAssetIds: readonly [AssetId, AssetId]
+  underlyingAssetRatios: readonly [string, string]
```

Comment:
can we get some nice comments on this one please? "for 1 LP token, this represents n and m units respectively of the underlying tokens in base units"

Reply:
https://github.com/shapeshift/web/pull/3195/commits/336ae080281a4d7c8da692ed04ba274ac8069f25